### PR TITLE
tests: fix "No space left on device" issue on amazon-linux

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The "|| true" is hiding a problem when we umount /tmp which is causing the error "No space left on device" when /tmp is used by another test. 

This is an example about how to reproduce it (I commented the umount time): https://paste.ubuntu.com/p/zsrqFDgD4V

This is an actual error reproduced on a travis execution:
https://api.travis-ci.org/v3/job/476382472/log.txt

The error can be reproduced by running the following tests (regression/lp-1800004, install-store-laaaarge and main/xdg-open) and the error is the following: https://travis-ci.org/snapcore/snapd/jobs/477104762#L1064

The fix is retrying the umount until the device is not busy anymore.

### CODE REVIEW

File: tests/main/install-store-laaaarge/task.yaml [hunk 1]
```diff
@@ -13,7 +13,17 @@ restore: |
     . "$TESTSLIB/systemd.sh"
 
     systemd_stop_units snapd.service snapd.socket
-    umount /tmp || true
```
Comment (mvo5) [inline tests/main/install-store-laaaarge/task.yaml]: Could we do a `umount --lazy /tmp` here? This might be sufficient to fix the issue.

Comment (bboozzoo) [reply tests/main/install-store-laaaarge/task.yaml]: BTW. I recall a note that `--lazy` does not work on 14.04 and you need to use `-l` instead.

Comment (sergiocazzolato) [reply tests/main/install-store-laaaarge/task.yaml]: Sure, I'll make the change and test it.

**Comment by mvo5** [comment]
Thank you @sergiocazzolato

**Pull request merged**